commons/passive/google/google_sleep_segment_event.avsc

```diff
@@ -0,0 +1,19 @@
+{
+  "namespace": "org.radarcns.passive.google",
+  "type": "record",
+  "name": "GoogleSleepSegmentEvent",
+  "doc": "Represents the result of sleep data after the user is awake.",
+  "fields": [
+    { "name": "time", "type": "double", "doc": "Device timestamp in UTC (s)." },
+    { "name": "timeReceived", "type": "double", "doc": "Device receiver timestamp in UTC (s)." },
+    { "name": "sleepStartTime", "type": "double", "doc": "The UNIX epoch time (s) for the moment when the user goes to sleep." },
```

Comment:
You can use the `time` variable for `startTime` instead. Then you can remove this field. In the doc of time, note that the time is when the user goes to sleep.